packages/storybook/stories/MessageThread/snippets/MessageWithFile.snippet.tsx

```diff
@@ -0,0 +1,45 @@
+import { FluentThemeProvider, MessageStatus, MessageThread } from '@azure/communication-react';
```

Comment:
I realize we use helper functions to render message thread messages.
However, I wanted these messages to show inline in storybook `show code` panel so that devs have an idea of how the file attachments are provided to message thread messages.